package.json

```diff
@@ -49,6 +49,7 @@
     "@michaellangbein/jsonix": "3.0.1-SNAPSHOT-3",
     "@schematics/angular": "^10.0.8",
     "@webcomponents/webcomponentsjs": "^2.4.4",
+    "atob": "^2.1.2",
```

Comment:
Do we need this dependency, normally it should be on every browser (https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob) and in Node you can decode base64 with Buffer.

Reply:
>  in Node you can decode base64 with Buffer
Didn't know about that - thanks!! Will adjust things accordingly.

Reply:
Alright, atob has been removed: https://github.com/dlr-eoc/ukis-frontend-libraries/pull/49/commits/6dc6c0c0f3ed0f2ef4164e2959d0b18152814e39